requirements.txt

```diff
@@ -1,120 +1,115 @@
-azure-batch==4.1.3
-azure-common==1.1.21
-azure-cosmosdb-nspkg==2.0.2
-azure-cosmosdb-table==1.0.5
-azure-datalake-store==0.0.45
-azure-eventgrid==1.3.0
-azure-graphrbac==0.40.0
-azure-keyvault==1.1.0
-azure-loganalytics==0.1.0
-azure-mgmt==4.0.0
-azure-mgmt-advisor==1.0.1
-azure-mgmt-applicationinsights==0.1.1
-azure-mgmt-authorization==0.50.0
-azure-mgmt-batch==5.0.1
-azure-mgmt-batchai==2.0.0
-azure-mgmt-billing==0.2.0
-azure-mgmt-cdn==3.1.0
-azure-mgmt-cognitiveservices==3.0.0
-azure-mgmt-commerce==1.0.1
-azure-mgmt-compute==4.6.2
-azure-mgmt-consumption==2.0.0
-azure-mgmt-containerinstance==1.5.0
-azure-mgmt-containerregistry==2.8.0
-azure-mgmt-containerservice==4.4.0
-azure-mgmt-cosmosdb==0.4.1
-azure-mgmt-datafactory==0.6.0
-azure-mgmt-datalake-analytics==0.6.0
-azure-mgmt-datalake-nspkg==3.0.1
-azure-mgmt-datalake-store==0.5.0
-azure-mgmt-datamigration==1.0.0
-azure-mgmt-devspaces==0.1.0
-azure-mgmt-devtestlabs==2.2.0
-azure-mgmt-dns==2.1.0
-azure-mgmt-eventgrid==1.0.0
-azure-mgmt-eventhub==2.6.0
-azure-mgmt-hanaonazure==0.1.1
-azure-mgmt-iotcentral==0.1.0
-azure-mgmt-iothub==0.5.0
-azure-mgmt-iothubprovisioningservices==0.2.0
-azure-mgmt-keyvault==1.1.0
-azure-mgmt-loganalytics==0.2.0
-azure-mgmt-logic==3.0.0
-azure-mgmt-machinelearningcompute==0.4.1
-azure-mgmt-managementgroups==0.1.0
-azure-mgmt-managementpartner==0.1.1
-azure-mgmt-maps==0.1.0
-azure-mgmt-marketplaceordering==0.1.0
-azure-mgmt-media==1.0.0
-azure-mgmt-monitor==0.5.2
-azure-mgmt-msi==0.2.0
-azure-mgmt-network==2.7.0
-azure-mgmt-notificationhubs==2.1.0
-azure-mgmt-nspkg==3.0.2
-azure-mgmt-policyinsights==0.1.0
-azure-mgmt-powerbiembedded==2.0.0
-azure-mgmt-rdbms==1.8.0
-azure-mgmt-recoveryservices==0.3.0
-azure-mgmt-recoveryservicesbackup==0.3.0
-azure-mgmt-redis==5.0.0
-azure-mgmt-relay==0.1.0
-azure-mgmt-reservations==0.2.1
-azure-mgmt-resource==2.2.0
-azure-mgmt-scheduler==2.0.0
-azure-mgmt-search==2.1.0
-azure-mgmt-servicebus==0.5.3
-azure-mgmt-servicefabric==0.2.0
-azure-mgmt-signalr==0.1.1
-azure-mgmt-sql==0.9.1
-azure-mgmt-storage==2.0.0
-azure-mgmt-subscription==0.2.0
-azure-mgmt-trafficmanager==0.50.0
-azure-mgmt-web==0.35.0
-azure-nspkg==3.0.2
-azure-servicebus==0.21.1
-azure-servicefabric==6.3.0.0
-azure-servicemanagement-legacy==0.20.6
-azure-storage-blob==1.5.0
-azure-storage-common==1.4.2
-azure-storage-file==1.4.0
-azure-storage-queue==1.4.0
-boto3==1.9.151
-botocore==1.12.151
-certifi==2019.3.9
-cffi==1.12.3
-chardet==3.0.4
-cryptography==2.6.1
-docutils==0.14
-idna==2.8
-isodate==0.6.0
-isort==4.3.20
-jmespath==0.9.4
-mccabe==0.6.1
-msrest==0.6.6
-msrestazure==0.6.0
-oauthlib==3.0.1
-pbr==5.2.1
-pycodestyle==2.5.0
-pycparser==2.19
-PyJWT==1.7.1
-pylint==2.3.1
-python-dateutil==2.8.0
-python-http-client==3.1.0
-requests==2.22.0
-requests-oauthlib==1.2.0
-s3transfer==0.2.0
-sendgrid==6.0.5
-six==1.12.0
-stevedore==1.30.1
-typed-ast==1.3.5
-urllib3==1.24.3
-virtualenv==16.6.0
-virtualenv-clone==0.5.3
-virtualenvwrapper==4.8.4
-wrapt==1.11.1
+adal     

```

Comment:
This file shouldn't be change unless you added new packages. Also, the package's version has to be specified. (You can read about requirements.txt)

Reply:
Added package versions and removed deprecated Azure library (https://github.com/Azure/azure-sdk-for-python/issues/10646)